.circleci/config.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+jobs:
+  build:
+    docker:
+    - image: dialonce/nodejs:lts-carbon # the primary container, where your job's commands are run
+    steps:
+    - checkout # check out the code in the project directory
+    - run: apk add --no-cache make python
+    - run: make deps
+    - run: make test
+## Comment out one permission denied is fixed
+#    - run: make sonar
```

Comment:
I think you are lacking env variables here. That's why it was failing with some permission errors

Reply:
they are set up in the ENV section on circle ci page

Reply:
as the ones for SWS s3 which work